model/src/test/java/com/epam/esm/model/dao/impl/TagDaoImplTest.java

```diff
@@ -0,0 +1,59 @@
+package com.epam.esm.model.dao.impl;
+
+import com.epam.esm.model.dao.TagDao;
+import com.epam.esm.model.dao.entity.Tag;
+import com.epam.esm.model.dao.exception.DaoException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+
+import javax.sql.DataSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+@RunWith(JUnitPlatform.class)
+class TagDaoImplTest {
```

Comment:
Why do not test classes/method have access modifiers?

Reply:
fix